c7n/resources/dynamodb.py

```diff
@@ -402,28 +402,24 @@ def augment(self, clusters):
         return list(filter(None, _dax_cluster_tags(
             resources,
             self.manager.session_factory,
-            self.manager.executor_factory,
             self.manager.retry,
             self.manager.log)))
 
 
-def _dax_cluster_tags(tables, session_factory, executor_factory, retry, log):
+def _dax_cluster_tags(tables, session_factory, retry, log):
     client = local_session(session_factory).client('dax')
 
     def process_tags(r):
-        tags = []
         try:
-            tags = retry(
+            r['Tags'] = retry(
                 client.list_tags, ResourceName=r['ClusterArn'])['Tags']
+            return r
         except (client.exceptions.ClusterNotFoundFault,
                 client.exceptions.InvalidARNFault,
```

Comment:
somewhat unrelated: Does this error actually come up? Why would we be passing in an invalid arn if it's coming from the resource itself?

Reply:
yeah, it seems odd, was trying to preserve existing semantics where possible, but agreed its seems odd, and good to remove.